core/ir/aarch64/codec.h

```diff
@@ -57,4 +57,21 @@ encode_common(byte *pc, instr_t *i, decode_info_t *di);
 #define BITS(_enc, bitmax, bitmin) \
     ((((uint32)(_enc)) >> (bitmin)) & (uint32)MASK((bitmax) - (bitmin) + 1))
 
+#if !defined(DR_HOST_NOT_TARGET) && !defined(STANDALONE_DECODER)
+/* i3044 TODO: Vector length will be read from cpuinfo, e.g.
+ * opnd_size_from_bytes(proc_get_vector_length()));
+ * Setting to fixed size for now in order to pass unit tests.
+ */
+#    define OPSZ_SVE_VL opnd_size_from_bytes(dr_get_sve_vl() / 8)
```

Comment:
Both these #defines are the same: they call dr_get_sve_vl(). Do we need to separate them?

Reply:
The #define for this #if clause (running on AArch64 and SVE h/w) will change. It's the same as the other #define temporarily so that tests on non-SVE AArch64 h/w pass.